v_4_0_0/filemanager.go

```diff
@@ -0,0 +1,42 @@
+package v_4_0_0
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io/ioutil"
+	"path"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+// Files is map[string]string for files that we fetched from disk and then filled with data.
+type Files map[string]string
+
+func RenderFiles(filespath string, ctx interface{}) (*Files, error) {
+	files := Files{}
+	dirList, err := ioutil.ReadDir(filespath)
+	if err != nil {
+		return nil, microerror.Maskf(err, "Failed to read files dir: %s, error: %#v", filespath, err)
+	}
+
+	for _, dir := range dirList {
+		fileList, err := ioutil.ReadDir(path.Join(filespath, dir.Name()))
+		if err != nil {
+			return nil, microerror.Maskf(err, "Failed to read dir: %s, error: %#v", path.Join(filespath, dir.Name()), err)
+		}
+
+		for _, file := range fileList {
+			tmpl, err := template.ParseFiles(path.Join(filespath, dir.Name(), file.Name()))
+			if err != nil {
+				return nil, microerror.Maskf(err, "Failed to file: %s, error: %#v", path.Join(filespath, dir.Name(), file.Name()), err)
+			}
+			var data bytes.Buffer
+			tmpl.Execute(&data, ctx)
+
+			files[dir.Name()+"/"+file.Name()] = base64.StdEncoding.EncodeToString(data.Bytes())
+		}
+	}
```

Comment:
I'm not sure what do you intend to use this function for but [filepath.Walk](https://golang.org/pkg/path/filepath/#Walk) may be a better candidate.